### PR TITLE
docker compose converter

### PR DESCRIPTION
**Pull request by ryan-cahill**

### CODE REVIEW

**Comment by davidthor** [comment]
This is great! Could you change the command to the following?

```sh
$ architect init --from-compose "docker-compose.yml"
```

**Comment by ryan-cahill** [comment]
@davidthor any other thoughts here?

File: test/commands/init.test.ts [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+
+  test
+    .do(ctx => mockAuth())
+    .finally(() => sinon.restore())
+    .stdout({ print })
+    .stderr({ print })
+    .command(['init', '--help'])
+    .it('succinctly describes the init command', ctx => {
+      expect(ctx.stdout).to.contain('Initialize an architect component from an existing docker-compose file\n')
+    });
```
Comment (davidthor) [inline test/commands/init.test.ts:27]: A test for help text contents might be overkill. More likely to get in the way then to be helpful.

**Review by davidthor — APPROVED** [review]
Looks good!

**Comment by tjhiggins** [comment]
> @davidthor any other thoughts here?
On me to review. Remind me again on Monday if I don't get to it this weekend.

File: src/commands/init.ts [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+      description: 'Path where the component file should be written to',
+      default: 'architect.yml',
+    }),
+    account: flags.string({
+      char: 'a',
+    }),
+    name: flags.string({
+      char: 'n',
+    }),
+    'from-compose': flags.string({
```
Comment (tjhiggins) [inline src/commands/init.ts:36]: I could have sworn that dashes didn't work, but great if they do. However, we should probably be consistent with dash or underscore.

**Pull request merged**